### ISSUE TITLE
Consider a data set without transit

### ISSUE THREAD

**Issue by dnesbitt61** [valhalla/mjolnir#369]
Check file sizes with and without transit. Does it make sense to enable a data set without transit if the transit data significantly increases tile sizes?

**Comment by kevinkreiser** [comment]
yeah if we could put schedules and edgeinfos in a seprate file this might work pretty nicely
